mono/metadata/threads-types.h

```diff
@@ -532,9 +533,15 @@ typedef struct {
 		int il_offset;
 		int native_offset;
 		const char *guid;
+
+#ifndef MONO_PRIVATE_CRASHES
```

Comment:
`ifdef` in headers to be avoided -- bifurcates the ABI.

Reply:
This can increase the stack size by a lot, I thought it best to leave out of release builds.

Reply:
By 600 bytes? I guess.